pkg/metadata/sbcs.go

```diff
@@ -148,6 +148,17 @@ func SBCs() []SBC {
 
 			MinVersion: semver.MustParse("1.8.0-alpha.1"),
 		},
+		{
+			Name: "rock5b",
+
+			OverlayName:  "rock5b",
+			OverlayImage: "siderolabs/sbc-rockchip",
+
+			Label:         "Radxa ROCK 5B",
+			Documentation: "/talos-guides/install/single-board-computers/rock5b/",
```

Comment:
do we have docs here?

Reply:
Currently running the linter on the docs, PR is on its way. Will reference it.

Reply:
https://github.com/siderolabs/talos/pull/10139